datatypes/src/operations/image/colorizer.rs

```diff
@@ -211,6 +312,24 @@ impl Colorizer {
         }
     }
 
+    pub fn over_color(&self) -> RgbaColor {
+        match self {
+            Colorizer::LinearGradient { over_color, .. }
+            | Colorizer::LogarithmicGradient { over_color, .. }
+            | Colorizer::Palette { over_color, .. } => *over_color,
+            Colorizer::Rgba => None,
```

Comment:
Why is it possible to return `None` if the return type is not `Option`? :thinking: 

If this is a problem, just output transparent for Rgba.

Reply:
That was totally my fault, sorry